### PR TITLE
✨Add locations list

### PR DESCRIPTION
**Pull request by abgeorge7**
A list of locations at the Where Can I Get Tested page

### CODE REVIEW

**Comment by dankolbman** [comment]
It's probably good to wrap the content in a container to limit the max width.
I extended the base layout in another pr to add a prop to the `Layout` component to do this for docs-like pages:
https://github.com/dankolbman/covid-tested/blob/eb536d654be9dc304bb123c9b471a3fcb812caa8/site/src/components/layout.js

File: site/src/pages/where-should-i-test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Locations from '../components/locations';
+import Layout from '../components/layout';
+import SEO from '../components/seo';
+import { fetchWithHeaders } from '../utils/api';
+
+class WhereShouldITestPage extends React.Component {
```
[the remark this answers is not in the document]
Comment (abgeorge7) [reply site/src/pages/where-should-i-test.js]: what about `state` ?  thought functional components were stateless

[the remark this answers is not in the document]
Comment (abgeorge7) [reply site/src/pages/where-should-i-test.js]: ah reading about hooks - I'm behind the times 👍

File: site/src/components/locations.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Locations = props => (
+  <div>
+    {props.locations.map(location => (
+      <div className="locations" key={location.name}>
+        <h2>{location.name}</h2>
+        <p>
+          <b>Address:</b> {location.street_address}
```
Comment (dankolbman) [inline site/src/components/locations.js]: Can we hide any fields that are null?

**Comment by abgeorge7** [comment]
Taken care of in #57

**Pull request closed without merging**